src/coreComponents/physicsSolvers/multiphysics/PoroelasticSolverEmbeddedFractures.cpp

```diff
@@ -832,7 +832,7 @@ void PoroelasticSolverEmbeddedFractures::updateState( DomainPartition & domain )
     arrayView1d< real64 const > const & deltaPressure =
       subRegion.getReference< array1d< real64 > >( FlowSolverBase::viewKeyStruct::deltaPressureString() );
 
-    forAll< serialPolicy >( subRegion.size(), [&] ( localIndex const k )
+    forAll< serialPolicy >( subRegion.size(), [=, &contactRelation] ( localIndex const k )
```

Comment:
@CusiniM this is the bug I mentioned to you.

Reply:
yeah, I had seen this and corrected it in my PR but it does not seem to be the only one...